rdrf/rdrf/routing/login_router.py

```diff
@@ -32,7 +33,9 @@ def get(self, request):
         redirect_url = None
 
         if user.is_authenticated:
```

Comment:
if settings.PROMS_SITE:
    if user.is_superuser:
        redirect to _HOME_PAGE 
    else:
         redirect to error or logout
 else:
     same as usual

Reply:
Necessary changes are made